docs/resources/server/enchantments.md

```diff
@@ -0,0 +1,85 @@
+# Enchantments
+
+Enchantments are enchancements that can be applied to tools and other items. As of 1.21, enchantments are stored on items as [Data Components], are defined in JSON, and are comprised of Enchantment Components.
+
+A new enchantment can be added by creating a JSON file in your namespace's `enchantment` datapack subfolder. For example, to create an enchantment called `examplemod:example_enchant`, one could create the file `data/examplemod/enchantment/example_enchantment.json`. Enchantments added this way are automatically registered into the game as part of the data pack loading process.
+
+The JSON format for Enchantments is summarized on the [Enchantment definition Minecraft wiki page].
+
+## Enchantment Components
+Enchantment Components are specially-registered [Data Components] that determine what an enchantment does. Enchantment Component Types must be [registered] to `BuiltInRegistries.ENCHANTMENT_EFFECT_COMPONENT_TYPE`.
+:::note
+There are many already existing Enchantment Component Types in vanilla that can be used to create enchantment logic.
+:::
```

Comment:
What are they? How can they be used?

Reply:
Is it important to replicate this information here from the Minecraft Wiki?

Reply:
They should be written for the intended audience of a modder. While it may have similar information to the Minecraft wiki, it'll probably be a bit more descriptive here, especially since this should explain how to create your own component types.